internal/sync/sync.go

```diff
@@ -71,12 +71,17 @@ func NewSynchronizer(cfg *config.Sync, feederClient *feeder.Client, syncManager
 	synchro := new(Synchronizer)
 	synchro.logger = Logger.Named("Sync Service")
 	synchro.feeder = feederClient
-	if !cfg.Trusted {
+	trusted := cfg.EthNode == ""
+	if trusted {
+		synchro.logger.Info("Defaulting to syncing from gateway")
+	} else {
 		ethereumClient, err := ethclient.Dial(cfg.EthNode)
-		if err != nil {
-			synchro.logger.Fatal("Unable to connect to Ethereum Client", err)
+		if err == nil {
+			synchro.l1Client = ethereumClient
+		} else {
+			synchro.logger.With("error", err).Error("Cannot connect to Ethereum Client: defaulting to syncing from gateway")
+			trusted = true
 		}
```

Comment:
I think we then want it to be fatal here because it means the user provided an Ethereum node address but that did not work.